python/dwave_qbsolv/dimod_wrapper.py

```diff
@@ -17,9 +17,13 @@ class QBSolv(dimod.TemplateSampler):
         >>> list(response.energies())
         '[1.0]'
     """
+    properties = None
+    parameters = None
+
+    def __init__(self):
+        self.properties = {}
+        self.parameters = {}
```

Comment:
This should be populated with the accepted parameters e.g. `num_repeats`, `seed`, ...

Should be a dict of the form `{'num_repeats': [ ], ...}` where the values are lists of the properties relevant to the parameter. So if the sampler has `properties` `{'max_num_repeats': 10000}` then `parameters` should be `{'num_repeats': ['max_num_repeats'], ...}`